docs/methods.md

```diff
@@ -24,6 +24,7 @@ Explore the complete public documentation for `py42` below.
 * [Security Data](methoddocs/securitydata.md)
 * [Legal Hold](methoddocs/legalhold.md)
 * [Departing Employee](methoddocs/departingemployee.md)
+* [High Risk Employee Lens](methoddocs/high_risk_employee.md)
```

Comment:
I think we are avoiding calling them `lens` in the SDK since that is more of a branding term and could go away. We are sticking to `detection lists`. Also, it should follow the same convention / format as the `Departing Employee` item listed right above it, unless they get consolidated (see my other PR comments). 

I think `High Risk Employee` would work, Code42 users should know what that is.

Reply:
Changed.